hpx/runtime/threads/coroutines/detail/context_linux_x86.hpp

```diff
@@ -140,7 +151,19 @@ namespace hpx { namespace threads { namespace coroutines
 
             x86_linux_context_impl()
                 : m_stack(nullptr)
-            {}
+            {
+                segv_stack.ss_sp = valloc(SEGV_STACK_SIZE);
```

Comment:
Could we have that depending on a define such that is configurable through cmake?